content/docs/user-guide/env.md

```diff
@@ -0,0 +1,23 @@
+# Environment Variables
+
+List of environment variables to configure DVC behavior.
+
+- `DVC_EXP_AUTO_PUSH`: If `true`, run `dvc exp push` at the end of a successful
+  <abbr>experiment</abbr>.
+- `DVC_EXP_GIT_REMOTE`: The name of the Git remote to use in `DVC_EXP_AUTO_PUSH`
+  (for example, `origin`).
+- `DVC_NO_ANALYTICS`: If `true`, disables
```

Comment:
```suggestion
- `DVC_NO_ANALYTICS`: If set, disables
```

Reply:
@skshetry Why is it your preference? I kind of see the logic, but part of me feels it's better to give an example of a valid value for the env var.

Reply:
We used `DVC_NO_ANALYTICS` to take effect on any value because with this envvar, you only want to disable. And, tbh with these envvars, I can never figure out if it should be set as 0/1/true/yes/no.

I am fine with showing an example. I wasn't sure if this was a reference or an example.